### PR TITLE
Fixes #1

### LINKED ISSUE #1: Multiple values per command are not correctly handled.
As per SVG specifications, a command can be followed by more than a set of values, for instance, the `M` command can be followed by 2n vales, not only by just two values.

### PR DESCRIPTION
**Pull request by mapio**
A simple patch of `_tokenize_path` solves #1.

An example is added (with a path containing multiple values per command), to motivate the patch.

### CODE REVIEW

**Comment by nvictus** [comment]
Thank you very much for this patch! However, it seems to break in the case of arc commands.

```
$ nosetests
E
======================================================================
ERROR: test_parser.test_parse_path
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/TEST/lib/python3.3/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/nvictus/svgpath2mpl/tests/test_parser.py", line 11, in test_parse_path
    path = parse_path(d)
  File "/home/travis/build/nvictus/svgpath2mpl/svgpath2mpl.py", line 229, in parse_path
    rx, ry, phi, large, sweep, x2, y2 = values
ValueError: need more than 0 values to unpack
----------------------------------------------------------------------
```

File: svgpath2mpl.py [hunk 1]
```diff
@@ -27,19 +27,30 @@
...
+    'M' : 2, # moveto
+    'L' : 2, # line
+    'H' : 1, # shorthand for horizontal line
+    'V' : 1, # shorthand for vertical line
+    'Q' : 4, # quadratic bezier
+    'T' : 4, # shorthand for smooth quadratic bezier
+    'C' : 6, # cubic bezier
+    'S' : 6, # shorthand for smooth cubic bezier
+    'Z' : 0, # closepath
+    'A' : 0  # arc
```
Comment (nvictus) [inline svgpath2mpl.py]: I think the problem is here. ~~The number of arc instructions is dynamically determined.~~ The number of arc parameters is 7: https://www.w3.org/TR/SVG/paths.html#PathDataEllipticalArcCommands

**Comment by nvictus** [comment]
Looks like 7c2053b solved it. Merging.

**Comment by mapio** [comment]
Thank you for accepting the pull request and sorry for having overlooked the tests!

**Pull request merged**